### ISSUE TITLE
Touch Screen

### ISSUE THREAD

**Issue by xujianing12** [efroemling/ballistica#508]
@efroemling Can you add touch screen controller on windows?
I can't run circle on windows. And I don't have Android Smart Phone : (

**Comment by Blaze-Corderz** [comment]
This is something i have been looking for!

**Comment by MatiasMGz** [comment]
Yeah, this would be a good addition to the game. Looking for this!
